app/views/locations/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>New Location</h1>
+
+<%= render 'form', location: @location %>
```

Comment:
Maybe pass in string for submit button, related to #99

Reply:
We don't understand the problem here.
The issue #99 exist for people but we can't reproduce it for locations.